src/imitation/rewards/serialize.py

```diff
@@ -30,7 +30,14 @@ def loader(path: str, venv: VecEnv) -> common.RewardFn:
         """Load train (shaped) or test (not shaped) reward from path."""
         del venv  # Unused.
         net = th.load(str(path))
-        reward = net.predict_reward_train if shaped else net.predict_reward_test
+        if not shaped and hasattr(net, "base"):
```

Comment:
Hm -- it feels weird that this is even part of the serialization logic, but we can perhaps clean this up in a separate PR.